### PR TITLE
Adding help to kde method

### PR DESCRIPTION
**Pull request by jaksle**
This is a simple commit which adds documentation to the `kde` method.

### CODE REVIEW

File: src/univariate.jl [hunk 1]
```diff
@@ -147,6 +147,20 @@ function conv(k::UnivariateKDE, dist::UnivariateDistribution)
...
 # main kde interface methods
+
+"""
+
+	kde(data; [options])
+	kde((xdata, ydata); [options])
+	
+Kernel density estimation method. Returns 1D or 2D KDE object. The grid used and the values of the estimated density can be obtained from fields `.x` and `.density` respectively. To obtain kde values at points different than the initial grid use the `pdf` method.
+
+The optional keyword arguments are
```
Comment (tpapp) [inline src/univariate.jl:158]:
```suggestion
The keyword arguments are
```

File: src/univariate.jl [hunk 2]
```diff
@@ -147,6 +147,20 @@ function conv(k::UnivariateKDE, dist::UnivariateDistribution)
 end
 
 # main kde interface methods
+
+"""
+
+	kde(data; [options])
+	kde((xdata, ydata); [options])
```
Comment (tpapp) [inline src/univariate.jl:154]:
```suggestion
	kde(data; kwargs...)
	kde((xdata, ydata); kwargs...)
```

**Comment by jaksle** [comment]
Could I ask to merge it? It is a simple PR so I think it should be straightforward.

The reason is I made a mistake and committed this PR from my master. So it blocks me from making any other PR, and I would like to fix the interface as discussed in issue #120.

**Comment by tpapp** [comment]
Merging, thanks.

**Comment by rashidrafeek** [comment]
Can a patch release be made here. It would help to have the documentation in the REPL.

**Comment by tpapp** [comment]
Thanks for the ping, I released a patch version.

**Pull request merged**